challenges.tex

```diff
@@ -0,0 +1,530 @@
+are not counting towards the scoring of the competition are carried out on
+a \SI{7 x 8}{\metre} field instead (corresponding to halve the field of the
+regular \ac{RCLL} field).
+The field is a subset of the magenta-halve of the regular field as depicted
+in \reffig{fig:challenge-field}.
+
+\begin{figure}
+	\centering
+	\begin{tikzpicture}[x=-1cm,y=1cm]
+	\fill[gray!50!white] (2,0) rectangle (5,1);
+	\node at (3.5,0) [anchor=south, font=\tiny, inner sep=1pt] {Insertion Zone};
+	\foreach \i in {0,...,5} {
+		\draw (0,\i) -- (5,\i);
+		\draw (\i,0) -- (\i,5);
+	}
+	\foreach \i in {1,...,3} {
+		\node at (1.5+\i,0.5) [circle,fill=magenta, draw=black,inner sep=0.2cm] {};
+	}
+	\foreach \x in {1,...,5} {
+		\foreach \y in {1,...,5} {
+			\node[font=\tiny,anchor=north west, inner sep=1pt] at (\x,\y) {M\_Z\x\y};
+		}
+	}
+	\node at (0,0) [circle,fill,inner sep=1.5pt,label={[font=\scriptsize]-45:$(0,0)$}]{};
+	\node at (5,0) [circle,fill,inner sep=1.5pt,label={[font=\scriptsize]225:$(-5,0)$}]{};
+	\node at (5,5) [circle,fill,inner sep=1.5pt,label={[font=\scriptsize]135:$(-5,5)$}]{};
+	\node at (0,5) [circle,fill,inner sep=1.5pt,label={[font=\scriptsize]45:$(0,5)$}]{};
+	\draw[line width=2pt,red,->] (0,0) -- node[above left,pos=1] {$x$} (-1.5,0);
+	\draw[line width=2pt,red,->] (0,0) -- node[below left,pos=1] {$y$} (0,1.5);
+\end{tikzpicture}
+	\caption{The \SI{5 x 5}{\metre} playing field.}
+	\label{fig:challenge-field}
+\end{figure}
+
+The entire area belongs to a single team. The bottom left-most
+\SI{1 x 3}{\metre} area is called the insertion zone and provides the starting
+positions for up to three robots.
+Contrary to the full RCLL field, it is not necessary to build partial walls
+around the insertion zone and robots may be move freely in and out of the
+insertion zone.
+\subsection{Mockup Machines}\label{sec:machines}
+In case no real \ac{MPS} stations are available, replications
+(so called \emph{mockup machines}) may be used, that do not need to
+physically perform the respective production steps. Instead, that work may
+be carried out by a human supervisor (see \refsec{sec:operators}).
+The minimum requirements for a mockup machine are specified in the following.
+
+Mockup machines are required to have the same box-like base-shape as specified
+in the RCLL rulebook.
+Additionally, the following parts need to be mounted:
+\begin{itemize}
+	\item a model of the conveyor belt
+	\item a shelf on the front right side of the box on stations replicating a \ac{CS}
+	\item either a shelf, a slide or a conveyor belt on the front right side
+of the box, such that it is accessible from the front on stations replicating a \ac{RS}
+\end{itemize}
+Models for a conveyor belt, shelf and slides can be found in the RCLL rulebook
+repository\footnote{\url{https://github.com/robocup-logistics/rcll-rulebook/tree/common/challenges/mock_up_models}}.
+
+The building materials for the models must be opaque, but may have any color.
```

Comment:
Hos close do the mock up conveyor/shelf/slide have to be to that model? Since it is not possible to 3D print them as a whole. Must it be exact or are those files just meant to be an orientation help?

Reply:
Since it is hard to exactly determine how close your model is to the real thing, as long as it looks like a slide, conveyor, shelf on the recording it will be fine.
If you need a precise model for your own software and hardware setup you free to make it as precise as you want.

Reply:
ok, thanks.